Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN apt-get update && apt-get install -y clang --no-install-recommends \
     && rm -rf /var/lib/apt/lists/*
 
 
-FROM ubuntu:16.04
+FROM debian:latest
```

Comment:
This way both images will share the base (less bandwidth/disk space usage)

Reply:
For me this actually increases the target image size:

```
# with f4984477ea1a being the new image
$ docker image inspect f4984477ea1a | grep Size
        "Size": 379804177,
        "VirtualSize": 379804177,
```

vs

```
docker image inspect b484c5bdad48 | grep Size
        "Size": 363301656,
        "VirtualSize": 363301656,
```

It’s a tradeoff, but it doesn’t seem to make much of a difference in the end. What do you think?